src/Parallel-Solvers/Implicit-Lagrange/FEA_Physics_Modules/FEA_Module_Elasticity.cpp

```diff
@@ -6255,9 +6255,51 @@ void FEA_Module_Elasticity::linear_solver_parameters(){
   }
   else{
     Linear_Solve_Params = Teuchos::rcp(new Teuchos::ParameterList("MueLu"));
-    std::string xmlFileName = "elasticity3D.xml";
-    //std::string xmlFileName = "simple_test.xml";
-    Teuchos::updateParametersFromXmlFileAndBroadcast(xmlFileName, Teuchos::Ptr<Teuchos::ParameterList>(&(*Linear_Solve_Params)), *comm);
+    if(module_params->muelu_parameters_xml_file){
+      std::string xmlFileName = module_params->xml_parameters_file_name;
+      //std::string xmlFileName = "simple_test.xml";
+      Teuchos::updateParametersFromXmlFileAndBroadcast(xmlFileName, Teuchos::Ptr<Teuchos::ParameterList>(&(*Linear_Solve_Params)), *comm);
+    }
+    else{
+      //set default parameters for MueLu
```

Comment:
How robust is the documentation for MeuLu?  If its good, I recommend we put a link in as a comment to the docs so users have a reference for what they can change.  If its not good, then ¯\_(ツ)_/¯

Reply:
Actually, it probably should not go here, unless we put the same comment in the other sections.  As long as it's somewhere obvious, it would be helpful.

Reply:
If in regards to the xml parameters, I would just use the examples in the MueLu package folder as guidance since thats whats up to date.